st2common/packaging/debian/postinst

```diff
@@ -3,6 +3,9 @@
 if [ -d /etc/st2 ]; then
   sed -i -r "s~(st2.*)/conf~/etc/\1~g" /etc/st2/st2.conf
   sed -i "s~vagrant~/home/stanley~g" /etc/st2/st2.conf
+
+  # Enable auth by default
+  sed -i "s~enable = False~enable = True~g" /etc/st2/st2.conf
```

Comment:
I'm not a fan of this approach - I prefer to be explicit. Doing that inside postinst step makes it kinda invisible...

I was thinking about having 2 configs - `conf/st2.dev.conf` and `conf/st2.prod.conf` and we use prod one for packages. @DoriftoShoes wdyt?

This way it's more explicit and visible